src/java/org/apache/cassandra/tools/nodetool/CompressionDictionary.java

```diff
@@ -18,26 +18,44 @@
 package org.apache.cassandra.tools.nodetool;
 
 import java.io.PrintStream;
+import java.util.List;
+import java.util.ArrayList;
 import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+import javax.management.openmbean.CompositeData;
+import javax.management.openmbean.TabularData;
 
 import com.google.common.util.concurrent.Uninterruptibles;
 
+import org.apache.cassandra.db.compression.CompressionDictionaryDetailsTabularData;
+import org.apache.cassandra.db.compression.CompressionDictionaryDetailsTabularData.CompressionDictionaryPojo;
 import org.apache.cassandra.db.compression.ICompressionDictionaryTrainer.TrainingStatus;
 import org.apache.cassandra.db.compression.TrainingState;
+import org.apache.cassandra.io.util.File;
+import org.apache.cassandra.io.util.FileUtils;
 import org.apache.cassandra.tools.NodeProbe;
+import org.apache.cassandra.tools.nodetool.formatter.TableBuilder;
 import org.apache.cassandra.utils.Clock;
+import org.apache.cassandra.utils.JsonUtils;
 import picocli.CommandLine.Command;
 import picocli.CommandLine.Option;
 import picocli.CommandLine.Parameters;
 
+import static java.nio.file.StandardOpenOption.CREATE;
+import static java.nio.file.StandardOpenOption.TRUNCATE_EXISTING;
+import static java.nio.file.StandardOpenOption.WRITE;
+
 @Command(name = "compressiondictionary",
-         description = "Manage compression dictionaries",
-         subcommands = { CompressionDictionary.Train.class })
+description = "Manage compression dictionaries",
+subcommands = { CompressionDictionary.TrainDictionary.class,
+                CompressionDictionary.ListDictionaries.class,
+                CompressionDictionary.ExportDictionary.class,
+                CompressionDictionary.ImportDictionary.class })
 public class CompressionDictionary
```

Comment:
@yifan-c I would rename this to `CompressionDictionaryCommand` or similar, maybe `CompressionDictionartyCommandGroup`.

There is already` CompressionDictionary` on server side and it is confusing to differentiate between each other when coding. I struggle to find the current one when they have same names, even they are in different packages.